utilities/generate_messages_catalog.py

```diff
@@ -97,15 +100,15 @@ def _log_function(item):
 
 
 FUNC_HANDLER = {
-    "info": ("info", 0),
-    "warning": ("warning", 0),
-    "error": ("error", 0),
-    "exception": ("exception", 0),
+    "info": lambda x: ("info", 0),
+    "warning": lambda x: ("warning", 0),
+    "error": lambda x: ("error", 0),
+    "exception": lambda x: ("exception", 0),
```

Comment:
could you give a brief explanation of these lambda's? I'm wondering why you need them when you aren't using ```x``` in them. or could you just git rid of ```x```?